BSDmakefile

```diff
@@ -0,0 +1,5 @@
+default:
```

Comment:
Since it doesn't work for Solaris, I think leaving 'You might need to use gmake' as a documentation issue is a better strategy for now. 2nd best would be to get our Makefile to give a reasonable error if called with something other than GNU Make but that's probably tricky. 

There is https://gist.github.com/jj1bdx/8cca44b363362cd25637 but I'm not it's worth rearranging Makefile right now to do that.